lib/purchases_flutter.dart

```diff
@@ -985,6 +986,21 @@ class Purchases {
     );
   }
 
+  /// Displays the specified store in-app message types to the user if there are any available to be shown.
+  /// - Important: This should only be used if you disabled these messages from showing automatically
+  /// during SDK configuration setting ``shouldShowInAppMessagesAutomatically`` to ``false``.
+  ///
+  /// @param [types] The types of messages to show.
+  static Future<void> showInAppMessages({
+    Set<InAppMessageType>? types,
+  }) =>
+      _channel.invokeMethod(
+        'showInAppMessages',
+        {
+          'types': types?.map((e) => e.index).toList(),
```

Comment:
Here we're converting the set to a list, which makes sense, but we should make sure we parse the list in objc.